core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1318,7 +1322,8 @@ private void deleteFromEntry(DeleteFileEntry entry) {
   private void deleteRecursiveInternal(LockedInodePath inodePath, boolean replayed, long opTimeMs)
```

Comment:
While it currently doesn't matter, I think this method should take a `DeleteOptions` instead of creating one internally.